pkg/systemstatsmonitor/disk_collector.go

```diff
@@ -300,6 +316,34 @@ func listRootBlockDevices(timeout time.Duration) []string {
 	return strings.Split(strings.TrimSpace(string(stdout)), "\n")
 }
 
+// recordBootDiskSize records the disk block devices that are not a slave or holder.
+func (dc *diskCollector) recordBootDiskSize(timeout time.Duration) {
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+
+	// "-d" prevents printing slave or holder devices. i.e. /dev/sda1, /dev/sda2...
+	// "-b" display the size in bytes.
+	// "-n" prevents printing the headings.
+	// "-p NAME" specifies to only print the device name.
+	cmd := exec.CommandContext(ctx, "lsblk", "-d", "-nb", "-o", "NAME,SIZE")
+	stdout, err := cmd.Output()
+	if err != nil {
+		glog.Errorf("Error calling lsblk")
+	}
+	lines := strings.Split(strings.TrimSpace(string(stdout)), "\n")
```

Comment:
suggest using `bufio.NewScanner` which has better memory performance (since we want to keep the NPD resource usage low)